### ISSUE TITLE
Compositing on Windows 7

### ISSUE THREAD

**Issue by barche** [K-3D/k3d#5]
Disabling compositing is no longer necessary on Win7. We should detect the version and only disable it on Vista.

**Comment by Megalomaniak** [comment]
8.0.1 disables it on windows 7. I'm assuming that's the intended behavior?

**Comment by barche** [comment]
Yes, I never got round to this. With the new pacman-based MSYS it may be easier to build Windows binaries from source, so maybe then this could be tackled.
